tests/uefi_sb/conftest.py

```diff
@@ -1,8 +1,11 @@
 import logging
 import pytest
 
-@pytest.fixture(scope='module')
+from packaging import version
+
+@pytest.fixture(scope='module') # FIXME: why not package scope?
 def pool_without_uefi_certs(host):
+    assert host.xcp_version < version.parse("8.3"), "fixture only relevant on XCP-ng 8.2"
```

Comment:
Why not use `host_less_than_8_3`?

Reply:
I haven't though of it, to be honest :)

However, there's a difference:
- `host_less_than_8_3` is designed so that we are not punished if we call a set of tests containing some that are not usable: the incompatible tests are just skipped.
- here I want that any developer who tries to use this fixture in a test that is going to run on XCP-ng 8.3+ gets an immediate error.